src/firefighter/slack/tasks/reminder_postmortem.py

```diff
@@ -72,3 +75,22 @@ def publish_fixed_next_actions(
         },
         client=client,
     )
+
+
+def send_reminder(incident: Incident, *, to_channel: bool) -> None:
```

Comment:
Document or make private: we not only need an incident, but an incident with `roles_prefetched`

Reply:
We could also make it explicit, by passing a `commander_role` argument, or something of this nature.